lang/src/org/partiql/lang/eval/Environment.kt

```diff
@@ -25,7 +25,7 @@ import java.util.TreeMap
  * @param session The evaluation session.
  * @param groups The map of [Group]s that is currently being built during query execution.
  */
-data class Environment(
+internal data class Environment(
```

Comment:
Currently the kdoc is "the environment for execution". Do you think is makes more sense to change it to "variables environment used during evaluation"?

Reply:
I think this version of the class is not *just* for variables--it includes `groups` too so the existing wording is more appropriate, IMHO.  Anyway, perhaps that is outside the scope of this CR.

Reply:
Then can we change it as "environment used during evaluation"? I think "execution" in the original definition is not really clear.